### PR TITLE
[ADD]Added website_product_brand module.

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: website_product_brand/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+{
+    'name': 'Product Brand Filtering in Website',
+    'category': 'Website',
+    'author': 'Serpent Consulting Services Pvt Ltd',
+    'website': 'http://www.serpentcs.com',
+    'summary': '',
+    'version': '1.0',
+    'description': """
+Product Brand Filtering in Website
```
Comment (pedrobaeza) [inline website_product_brand/__openerp__.py:31]: Please extract description to README.rst and follow OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: website_product_brand/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+import openerp.addons.website_sale.controllers.main
+from openerp import SUPERUSER_ID
+from openerp.addons.website.models.website import slug
+from openerp.addons.website_sale.controllers.main import table_compute, QueryURL
+
+PPG = 20
+PPR = 4
+
+
+class website_sale(openerp.addons.website_sale.controllers.main.website_sale):
```
Comment (pedrobaeza) [inline website_product_brand/controllers/main.py]: Use CamelCase for the class name.

File: website_product_brand/oca_dependencies.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+product-attribute
```
Comment (pedrobaeza) [inline website_product_brand/oca_dependencies.txt]: This file is put at root level, not here, and it's not a link, but a file with contents

**Comment by ghost** [comment]
@pedrobaeza 
Can you please help me in failing runbot testing ?
I have checked the logs in runbot, it says that product_brand module is not available on the system.
I had added module dependencies in oca_dependencies.txt file.

Regards.

File: website_product_brand/__openerp__.py (hunk 1 above)
Comment (ghost) [inline website_product_brand/__openerp__.py:31]: Added readme.rst as per OCA's template.

File: website_product_brand/controllers/main.py (hunk 2 above)
Comment (ghost) [inline website_product_brand/controllers/main.py]: Done.

File: website_product_brand/oca_dependencies.txt (hunk 3 above)
Comment (ghost) [inline website_product_brand/oca_dependencies.txt]: Added this file in root.

Can you please explain how to add file with contents ?

Thanks.

**Comment by rafaelbn** [comment]
hi @JayVora-SerpentCS , @pedrobaeza what is needed to finish this PR? We can finish this work if needed

**Comment by ghost** [comment]
Hello @rafaelbn 

We are facing a dependency issue.

We had checked the logs in runbot, it says that product_brand module is not available on the system.

We had added branch path in .travis.yml as well as in oca_dependecies.txt.

We would like to have the guidance on oca-dep.text

Thanks.

**Comment by ghost** [comment]
@oca

I have traced the travis log file, error is raised because of 'website_sale_vat_required' module.

Can you please help me to pass travis test ?

Thanks.

**Comment by pedrobaeza** [comment]
If you rebase, it should be fixed now

**Comment by ghost** [comment]
@pedrobaeza 

Thanks for reply.

I had rebased the branch.

**Comment by pedrobaeza** [comment]
Uhm, it's still there. I thought the problem was solved. @eLBati, isn't the problem with website_sale_vat_required solved?

**Comment by eLBati** [comment]
@pedrobaeza it is with https://github.com/OCA/website/pull/57

**Comment by pedrobaeza** [comment]
Ah, OK, can we move forward with the comments made to get a green test?

**Comment by ghost** [comment]
@pedrobaeza 

Can you advice me what next i have to do in order to get a green list ?

Thanks.

**Comment by pedrobaeza** [comment]
It's not possible yet until the other problem is solved. Let's push to get the branch green, and then you need to rebase.

**Comment by pedrobaeza** [comment]
Sorry for the noise, but we have recently discussed this (https://odoo-community.org/groups/oca-contributors-15/oca-contributors-24218) and finally decided that the best place is e-commerce, so I close this one.

Sorry again for bothering you.

**Pull request closed without merging**